src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -114,6 +111,24 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         /// </summary>
         internal FeatureFilterTracing FeatureFilterTracing { get; set; } = new FeatureFilterTracing();
 
+        /// <summary>
+        /// Options used to configure provider startup.
+        /// </summary>
+        internal StartupOptions Startup { get; set; } = new StartupOptions();
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AzureAppConfigurationOptions"/> class.
+        /// </summary>
+        public AzureAppConfigurationOptions()
+        {
+            _adapters = new List<IKeyValueAdapter>()
+            {
+                new AzureKeyVaultKeyValueAdapter(new AzureKeyVaultSecretProvider()),
+                new JsonKeyValueAdapter(),
+                new FeatureManagementKeyValueAdapter(FeatureFilterTracing)
```

Comment:
This was my solution to needing to pass `FeatureFilterTracing` to the `FeatureManagementKeyValueAdapter` constructor, since the property couldn't be referenced from outside a method.